llpc/context/llpcCompiler.cpp

```diff
@@ -845,10 +845,11 @@ Result Compiler::buildPipelineWithRelocatableElf(Context *context, ArrayRef<cons
     }
     // Note that this code updates m_pipelineHash of the pipeline context. It
     // must be restored before we link the pipeline ELF at the end of this for-loop.
-    auto hashForCacheLookup = convertToHashUsedForCacheLookup(cacheHash);
-    context->getPipelineContext()->setPipelineHashCode(hashForCacheLookup);
-    LLPC_OUTS("Hash for " << getShaderStageName(static_cast<ShaderStage>(stage)) << " stage cache lookup: "
-                          << formatBytesLittleEndian<uint8_t>(hashForCacheLookup.bytes) << "\n");
+    context->getPipelineContext()->setHashForCacheLookUp(cacheHash);
+    LLPC_OUTS("Finalized Hash for " << getShaderStageName(static_cast<ShaderStage>(stage)) << " stage cache lookup: "
```

Comment:
nit: why is 'Hash' capitalized?
Also note to self: this will break cache-creator tests.